docs/core/install/windows.md

```diff
@@ -82,6 +82,39 @@ The SDK is used to build and publish .NET apps and libraries. Installing the SDK
 > [!div class="button"]
 > [Download .NET SDK](https://dotnet.microsoft.com/download/dotnet)
 
+## ARM-based Windows PCs
```

Comment:
I switched to use Arm on the website with @richlander's advice.
https://www.arm.com/